modules/final_radiological_review/test/final_radiological_reviewTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * final_radiological_review automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+class finalRadiologicalReviewTestIntegrationTest extends LorisIntegrationTest
+{
+    /**
+     * Tests that, when loading the final_radiological_review module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testFinalRadiologicalReviewDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=final_radiological_review");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Final Radiological Review", $bodyText);
```

Comment:
Yes. If there's no subtest, it loads the menu filter, and if there is a subtest, it loads the form that allows you to edit the radiological review.

We probably can/should rename the subtest, but right now it's named the same thing.